container/features/src/main/resources/features-core.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<features name="opennms-core-${project.version}"
+          xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0"
+>
+
+    <!--
+         This file is for features that should be shared across Horizon, Minion, and Sentinel.
+         Note that _any_ bundle dependency that has classes exported in `custom.properties` must
+         have `dependency="true"` set, so that it won't try to re-load those classes on the
+         Karaf side.
+    -->
+
+    <feature name="cassandra-driver" version="${cassandraVersion}" description="Datastax Cassandra Driver">
```

Comment:
is this a new feature? It looks like the other features here were moved from other config files, but I didn't see this anywhere else.

Reply:
Yes, it overrides the (much older) one that's in the newts feature file (which is external to our own project).

Long term, this should move back into newts, but there's a [larger discussion to be had about that](https://issues.opennms.org/browse/NEWTS-115) elsewhere.